src/core/lib/security/authorization/sdk_server_authz_filter.cc

```diff
@@ -83,7 +79,7 @@ bool SdkServerAuthzFilter::IsAuthorized(
         engines.allow_engine->Evaluate(args);
     if (decision.type == AuthorizationEngine::Decision::Type::kAllow) {
       if (GRPC_TRACE_FLAG_ENABLED(grpc_sdk_authz_trace)) {
-        gpr_log(GPR_INFO, "chand=%p: request allowed by policy %s.", this,
+        gpr_log(GPR_DEBUG, "chand=%p: request allowed by policy %s.", this,
                 decision.matching_policy_name.c_str());
```

Comment:
I changed the log level to reduce the log traffic.
https://b.corp.google.com/issues/218495838